torch/utils/dlpack.py

```diff
@@ -36,7 +36,7 @@ class DLDeviceType(enum.IntEnum):
 def from_dlpack(ext_tensor: Any) -> torch.Tensor:
     """from_dlpack(ext_tensor) -> Tensor
 
-    Convers a tensor from a external library into a ``torch.Tensor``
+    Converts a tensor from a external library into a ``torch.Tensor``
```

Comment:
Can you drop this change? I have the same change in gh-70437, and not having it here avoids a conflict.